index.js

```diff
@@ -14,6 +14,7 @@
     .provider('Analytics', function () {
       var accounts,
           analyticsJS = true,
+          appName,
```

Comment:
Change this to hybridMobileAppName. That makes it clearly related to hybridMobileSupport and not a generic configuration value.